src/Schema/Builder.php

```diff
@@ -154,31 +157,36 @@ public function dropAllTables()
     }
 
     /** @param string|null $schema Database name */
-    public function getTables($schema = null)
-    {
+    private function getCollectionRows(
+        $schema = null,
+        ?string $collectionType = null,
```

Comment:
`$collectionType` is always set, you don't need to make it nullable.

```suggestion
        string $collectionType,
```

Reply:
True, although I was thinking it might be nice if we ever want to get everything. Bit premature probably.